iroha_actor/src/lib.rs

```diff
@@ -279,24 +288,6 @@ pub trait Actor: Send + Sized + 'static {
     {
         Self::default().start().await
     }
-
-    /// If this actor subscribes to some broker messages it needs to return broker instance here
-    /// For the subscribtions to be properly canceled on actor stop.
-    fn broker(&self) -> Option<&broker::Broker> {
-        None
-    }
-
-    /// At stop hook of actor. Generally should not be overriden. Calls `on_stop`.
-    async fn on_stop_internal(&mut self, ctx: &mut Context<Self>) {
-        if let Some(broker) = self.broker() {
-            ctx.unsubscribe_from_on_stop
-                .iter()
-                .for_each(|(subscription_id, message_type)| {
-                    broker.unsubscribe_by_type_id(*message_type, *subscription_id)
-                });
-        }
-        self.on_stop(ctx).await;
-    }
```

Comment:
How do you unsubscribe automatically without knowing the broker?

Reply:
Lazily. Just check manually via `is_closed`

Reply:
ok